### DISCUSSION TITLE
"Hide UI" shortcut

### DISCUSSION THREAD

**Discussion by aidanhadley** [Floorp-Projects/Floorp#550] [Feature Request]
Similar to Vivaldi's "UI Toggle" feature, this would remove all border decorations. Right now ive done a patch with some JavaScript (i think) and Custom Actions, but having this native would make it feel much more intuitive.

**Comment by gunir** [comment]
Read this post, you still need to learn about Toolbar ID, which is the basic things if you want to go further:
https://github.com/Floorp-Projects/Floorp/discussions/556#discussioncomment-7465366
